src/handlers/handlerUtils.ts

```diff
@@ -784,6 +810,16 @@ export async function tryTargetsRecursively(
     currentTarget.requestTimeout = inheritedConfig.requestTimeout;
   }
 
+  if (currentTarget.inputGuardrails) {
```

Comment:
Should we also handle scenario where user might have sent both: before_request_hooks and input_guardrails?